cvmfs/platform_linux.h

```diff
@@ -92,10 +89,8 @@ inline bool platform_umount(const char* mountpoint, const bool lazy) {
       return false;
     }
     FILE *fmntnew = setmntent(mntnew.c_str(), "w+");
-    if (!fmntnew &&
-        (chmod(mntnew.c_str(), mtab_info.st_mode) != 0) &&
-        (chown(mntnew.c_str(), mtab_info.st_uid, mtab_info.st_gid) != 0))
-    {
+    if (!fmntnew && (chmod(mntnew.c_str(), mtab_info.st_mode) != 0) &&
```

Comment:
I have clang-format running on save. Most of the time it's a time saver, but it's true that it creates some noise the first time it's applied on files.

Sometimes it's bad, like today when it broke compilation in this branch, since it reordered the include files alphabetically in a file where they needed to be in a specific order.